.github/workflows/dockerimage.yml

```diff
@@ -9,6 +9,7 @@ on:
   pull_request:
     paths:
       - Dockerfile
+      - .dockerignoree
```

Comment:
Looks like adding this doesn't trigger the docker-image action.

Reply:
We could do:

- Commit this change and an empty `.dockerignore`.
- Create a new PR and edit `.dockerignore` in that PR.

Whta do you think?

Reply:
It was just a typo :)